addons/hr_recruitment/views/hr_recruitment_views.xml

```diff
@@ -33,14 +33,16 @@
                 <field name="job_id"/>
                 <field name="name" readonly="1"/>
                 <field name="stage_id"/>
+                <field name="application_status" optional="show"/>
```

Comment:
```suggestion
                <field name="application_status" optional="show" attrs="{'invisible': [('application_status', '=', 'ongoing')]}"/>
```

Reply:
Ah nice I didn't know you could do it like this :)